rule/import_shadowing.go

```diff
@@ -113,3 +116,17 @@ func (w importShadowing) Visit(n ast.Node) ast.Visitor {
 
 	return w
 }
+
+func isVersion(name string) bool {
+	if len(name) < 2 || (name[0] != 'v' && name[0] != 'V') {
+		return false
+	}
+
+	for i := 1; i < len(name); i++ {
+		if name[i] < '0' || name[i] > '9' {
+			return false
+		}
+	}
+
+	return true
+}
```

Comment:
While the code seems OK, could you please add a test for v12345 as a future refactoring may break the support of vXX